lib/ticks-to-tree.js

```diff
@@ -25,7 +27,7 @@ function ticksToTree (ticks, mapFrames, inlined, binary) {
 
   // Spawn a file that throws an error to get the loading stack.
   // Then create a regular expression that matches all those files.
-  const child = spawnSync(binary, [join(__dirname, 'loading-stacks')])
+  const child = spawnSync(binary, ['--experimental-modules', join(__dirname, 'loading-stacks')])
```

Comment:
We should be working even on Node.js versions where this flag is not available, as this will error otherwise. How can we handle that?

Reply:
0x support is from 8.5 upwards and that is exactly the version where support for esm is added as well. That's why I did not add a special casing.

It would be easy to just check for the version otherwise.

Reply:
Thanks! As for future proofing it, I would recommend to check if the process failed, and then rerun without the flag.

Reply:
Done.

Reply:
Done.